### ISSUE TITLE
Accept mistakes

### ISSUE THREAD

**Issue by luchian** [pixeltrove/ignition-vanilla#22]
Coding and adjacent activities such as versioning may came about in a non-standard manner or lead to unwanted results. Guidelines and fixes are just a few of the ways in which slip-ups can be corrected. Document these practices in order to establish prescriptive actions.

**Comment by luchian** [comment]
Versioning should follow the guidelines established by the GitFlow workflow. _Feature_ branches emerge from the _develop_ branch and merge back into it when complete. When enough features have been implemented, a _release_ branch is created off of _develop_. Only fixes are allowed on the _release_ branch; once testing is complete the _release_ branch is merged into _master_ and _develop_. _Hotfix_ branches are used to create emergency fixes. They are branched directly from the _master_ branch and merge back into it and _develop_.
